lib/views/auth.html

```diff
@@ -1,14 +1,15 @@
 <h2>Authorize</h2>
 
-<div class="form">
+<div class="oauth-form">
   <p>The application <em><%= client_id %></em> hosted at
     <em><%= client_host %></em> wants to access these resources for user
```

Comment:
Shouldn't this be the other way around, because the ID here is actually the URL, while the host is used as the app name?